authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc

```diff
@@ -28,26 +27,64 @@ Cloud providers use different terms for their implementation of this authenticat
 |Google Cloud Platform (GCP)
 |GCP Workload Identity
 
-//|global Microsoft Azure
-//|Azure AD Workload Identity
-//
+|global Microsoft Azure
```

Comment:
Oh yeah good point in this context, I have been trying to keep it lower because it's not part of the capitalized product name, but this makes more sense capitalized :+1: